applications/crossbar/src/cb_apps_util.erl

```diff
@@ -298,33 +281,20 @@ get_plan_apps(ServicePlan) ->
 %%--------------------------------------------------------------------
 -spec find_enabled_apps(kz_json:object()) -> kz_json:objects().
 find_enabled_apps(PlanAppsJObj) ->
-    kz_json:foldl(
-      fun find_enabled_apps_fold/3
-                 ,[]
-                 ,PlanAppsJObj
-     ).
-
-%%--------------------------------------------------------------------
-%% @private
-%% @doc
-%% @end
-%%--------------------------------------------------------------------
--spec find_enabled_apps_fold(ne_binary(), kzd_item_plan:doc(), kz_json:objects()) -> kz_json:objects().
-find_enabled_apps_fold(AppName, PlanApp, Acc) ->
-    AppId = kz_json:get_value(<<"app_id">>, PlanApp),
-    case (not kzd_item_plan:is_enabled(PlanApp))
-        orelse find_app(AppId, PlanApp)
-    of
-        'true' ->
-            lager:debug("excluding app ~s(~s)", [AppName, AppId]),
-            Acc;
-        'undefined' ->
-            lager:debug("excluding app ~s(~s)", [AppName, AppId]),
-            Acc;
-        AppJObj ->
-            lager:debug("including app ~s(~s) from plan", [AppName, AppId]),
-            [kz_json:delete_key(<<"published">>, AppJObj)|Acc]
-    end.
+    [kz_json:delete_key(<<"published">>, MaybeAppJObj)
```

Comment:
@fenollp Howdy, this PR looks good except for this... I would not describe this as more readable.  Why the change to list comprehensions?